docs/50_eos-evm/30_developer-guide/30_deploy-local-nodes.md

```diff
@@ -1039,7 +1039,7 @@ Response:
 
 #### Setup Proxy To Separate Read Requests From Write Requests
 
-The proxy program will separate Ethereum's write requests (such as eth_sendRawTransaction,eth_gasPrice) from other requests (treated as read requests). The write requests should go to Transaction Wrapper (which wrap the ETH transaction into Antelope transaction and sign it and push to the Antelope blockchain). The read requests should go to TrustEVM-RPC.
+The proxy program will separate Ethereum's write requests (such as eth_sendRawTransaction,eth_gasPrice) from other requests (treated as read requests). The write requests should go to Transaction Wrapper (which wrap the ETH transaction into Antelope transaction and sign it and push to the Antelope blockchain). The read requests should go to EOS EVM RPC.
```

Comment:
to Transaction Wrapper -> to the Transaction Wrapper
which wrap -> which wraps
into Antelope transaction and sign it and -> into an Antelope transaction, sign it, and